assignment_1.rb

```diff
@@ -0,0 +1,30 @@
+#Blocks
+arr = [1,2,3,4,5,6,7,8,9,10]
+count=0
+arr.each do |item|
+  if((item % 2) == 0)
+  count=count + 1
```

Comment:
please check the indentation. 
Also, can you try this with {} block?

Reply:
Done using {} block